src/olympia/users/management/commands/process_users.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+
+from django.db.models import Q
+
+from olympia.amo.management import ProcessObjectsCommand
+from olympia.users.models import UserProfile
+from olympia.users.tasks import (
+    backfill_activity_and_iplog,
+)
+
+
+class Command(ProcessObjectsCommand):
+    def get_model(self):
+        return UserProfile
+
+    def get_tasks(self):
+        return {
+            # Don't forget to use the --with_deleted argument when triggering
+            # this task!
```

Comment:
why?

Reply:
I need to remove this - I wanted to make sure to run this even on deleted users, but `UserProfile` doesn't have `unfiltered`, so it will do that out of the box (in fact `--with_deleted` doesn't work)

Reply:
https://github.com/mozilla/addons-server/pull/19616/commits/f8fd73507f33a1acd6a99e1677c768a309a82d2d